src/fides/api/models/tcf_publisher_overrides.py

```diff
@@ -0,0 +1,23 @@
+from sqlalchemy import Boolean, Column, Integer, String, UniqueConstraint
+from sqlalchemy.ext.declarative import declared_attr
+
+from fides.api.db.base_class import Base
+
+
+class TCFPublisherOverride(Base):
+    """
+    Stores TCF Publisher Overrides
+
+    Allows a customer to override Fides-wide which Purposes show up in the TCF Experience, and
+    specify a global legal basis for that Purpose.
+    """
+
+    @declared_attr
+    def __tablename__(self) -> str:
+        return "tcf_publisher_overrides"
+
+    purpose = Column(Integer, nullable=False)
+    is_included = Column(Boolean, server_default="t", default=True)
+    required_legal_basis = Column(String)
+
+    UniqueConstraint("purpose")
```

Comment:
Assumed purposes, not special purposes were being toggled here, from the mockups.  We can't save preferences against a special purpose x legal basis, only purpose x legal basis, so I assume that's the distinction. Still, someone may want to suppress the special purpose altogether? In that case I might add another column here that clarifies if the record is for a purpose or special purpose.